CHANGELOG.md

```diff
@@ -17,6 +17,9 @@ If the change isn't user-facing but still relevant enough for a changelog entry,
 * (internal)? scope: short description (#pr, @author)
 -->
 
+### Added
+* anxcloud_kubernetes_cluster: optional fields in Kubernetes cluster schema for existing prefixes in which the cluster should be deployed (#123, @marioreggiori)
```

Comment:
I think we can strip the "in Kubernetes cluster schema" and maybe change "for existing prefixes" to "for using existing prefixes" and "... to deploy the cluster into"